src/app/shared/mappings-filter/mappings-filter.component.html

```diff
@@ -6,13 +6,13 @@
       </div>
     </ng-container>
     <input #searchInput [ngClass]="{'transparent': focusedOutcome && !focused}" type="search" placeholder="Search for mappings..." aria-label="Search for Mappings" (focus)="focusInput(searchInput.value)" (blur)="focused = false;" />
-    <div class="search-sources" id="search-sources" [ngClass]="{'active': source}" tip="Filter results by sources" tipLocation="bottom" (click)="sourcesDown = true; preventClose('sources')">
+    <button class="search-sources" id="search-sources" [ngClass]="{'active': source}" aria-label="Filter results by sources" tip="Filter results by sources" tipLocation="bottom" (click)="sourcesDown = true; preventClose('sources')" (keypress)="sourcesDown = true; preventClose('sources')">
```

Comment:
Can we remove the keypress directive here? That event will fire for ANY key that gets pressed, not just one of the activation keys (Enter or Space). The new activate directive should make this superfluous anyway 😄